### PR TITLE
fix for issue #16629

### PR DESCRIPTION
**Pull request by shub-asa1**
this is a fix for issue #16629 .

### CODE REVIEW

File: content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 1]
```diff
@@ -323,7 +323,7 @@ you can use to more precisely control the behavior of liveness and readiness
 checks:
 
 * `initialDelaySeconds`: Number of seconds after the container has started
-before liveness or readiness probes are initiated.
+before liveness or readiness probes are initiated.Defaults to 0 seconds.Minimum value is 0.
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md]: nit:
```suggestion
before liveness or readiness probes are initiated. Defaults to 0 seconds. Minimum value is 0.
```

Comment (shub-asa1) [reply content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md]: @sftim Thanks for your feedback.

**Pull request merged**